File: Python/0026_remove_duplicates_from_sorted_array/solution.py
def remove_duplicates(nums):
    replica = list()
    for num in nums:
        if num not in replica:
            replica.append(num)
    for _ in range(len(nums) - len(replica)):
        replica.append("")
    for i in range(len(replica)):
        nums[i] = replica[i] 
    final = list()
    for num in nums:
        if num != "":
            final.append(num)
    
    return [len(final),nums]

nums = [0,0,1,1,1,2,2,3,3,4]
print(remove_duplicates(nums))

File: JavaScript/0026_remove_duplicates_from_sorted_array/solution.js
function remove_duplicates(nums) {
    let replica = [];
    for (let num of nums) {
        if (!replica.includes(num)) {
            replica.push(num);
        }
    }
    let count = nums.length - replica.length;
    for (let i = 1; i <= count; i ++) {
        replica.push("");
    }
    for (let i = 0; i < replica.length; i ++) {
        nums[i] = replica[i];
    }
    let final = [];
    for (let num of nums) {
        if (num !== "") {
            final.push(num);
        }
    }
    return final.length;
}

const nums = [0,0,1,1,1,2,2,3,3,4];
console.log(remove_duplicates(nums));